### PR TITLE
HTTPS is better than HTTP

### PR DESCRIPTION
**Pull request by RyDroid**
```
find src/ -type f -print0 | \
  xargs -0 sed -i 's$http://phpmd.org$https://phpmd.org$g'
```

https://github.com/phpmd/phpmd/pull/509

### CODE REVIEW

File: src/test/resources/files/pmd/default-xml.xml [hunk 1]
```diff
@@ -5,4 +5,4 @@
       The function ccn_function() has a Cyclomatic Complexity of 12. The configured cyclomatic complexity threshold is 10.
     </violation>
   </file>
-</pmd>
\ No newline at end of file
+</pmd>
```
Comment (ravage84) [inline src/test/resources/files/pmd/default-xml.xml]: Could you revert that unrelated change, please?

File: src/test/resources/files/pmd/single-directory.xml [hunk 2]
```diff
@@ -18,4 +18,4 @@
       The method doSomething() has an NPath complexity of 50000. The configured NPath complexity threshold is 50.
     </violation>
   </file>
-</pmd>
\ No newline at end of file
+</pmd>
```
Comment (ravage84) [inline src/test/resources/files/pmd/single-directory.xml]: Could you revert that unrelated change, please?

File: src/test/resources/files/pmd/single-file.xml [hunk 3]
```diff
@@ -5,4 +5,4 @@
       The function ccn_function() has a Cyclomatic Complexity of 12. The configured cyclomatic complexity threshold is 10.
     </violation>
   </file>
-</pmd>
\ No newline at end of file
+</pmd>
```
Comment (ravage84) [inline src/test/resources/files/pmd/single-file.xml]: Could you revert that unrelated change, please?

**Comment by RyDroid** [comment]
I removed the final newlines (that were useful for `cat`).

**Comment by ravage84** [comment]
Thanks 🍸
